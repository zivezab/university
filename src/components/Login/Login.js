import React, { Component } from "react";
import { Button, Form } from "react-bootstrap";

class Login extends Component {
  render() {
    return (
      <Form>
        <Form.Group controlId="formBasicEmail">
          <Form.Label>Email address</Form.Label>
          <Form.Control required type="email" placeholder="Enter email" />
          <Form.Text className="text-muted"></Form.Text>
        </Form.Group>
        <Form.Group controlId="formBasicPassword">
          <Form.Label>Password</Form.Label>
          <Form.Control required type="password" placeholder="Password" />
        </Form.Group>
        <Form.Group controlId="formBasicCheckbox">
          <Form.Check type="checkbox" label="Remember Me" />
        </Form.Group>
        <span className="float-right">
          <Button variant="primary" type="submit">
            Login
          </Button>{" "}
          <Button variant="success" type="submit">
            Register
          </Button>
        </span>
      </Form>
    );
  }
}

export default Login;
