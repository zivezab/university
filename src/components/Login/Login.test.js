import { shallow } from "enzyme";
import toJSON from "enzyme-to-json";
import React from "react";

import Login from "./Login";

test("matches snapshot", () => {
  const wrapper = shallow(<Login />);

  expect(toJSON(wrapper)).toMatchSnapshot();
});
