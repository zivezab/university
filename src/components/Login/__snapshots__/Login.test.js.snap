// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<Form
  inline={false}
>
  <FormGroup
    controlId="formBasicEmail"
  >
    <FormLabel
      column={false}
      srOnly={false}
    >
      Email address
    </FormLabel>
    <FormControl
      placeholder="Enter email"
      required={true}
      type="email"
    />
    <FormText
      className="text-muted"
    />
  </FormGroup>
  <FormGroup
    controlId="formBasicPassword"
  >
    <FormLabel
      column={false}
      srOnly={false}
    >
      Password
    </FormLabel>
    <FormControl
      placeholder="Password"
      required={true}
      type="password"
    />
  </FormGroup>
  <FormGroup
    controlId="formBasicCheckbox"
  >
    <FormCheck
      label="Remember Me"
      type="checkbox"
    />
  </FormGroup>
  <span
    className="float-right"
  >
    <Button
      active={false}
      disabled={false}
      type="submit"
      variant="primary"
    >
      Login
    </Button>
     
    <Button
      active={false}
      disabled={false}
      type="submit"
      variant="success"
    >
      Register
    </Button>
  </span>
</Form>
`;
